kolibri/plugins/coach/assets/src/views/lessons/LessonSummaryPage/sidePanels/LessonResourceSelection/subPages/PreviewSelectedResources/PreviewContent.vue

```diff
@@ -0,0 +1,268 @@
+<template>
+
+  <div>
+    <div class="channel-header">
+      <div>
+        <h6>
+          {{ coreString('selectFromChannels') }}
+        </h6>
+      </div>
+
+      <div>
+        <template v-if="isSelected">
+          <KIcon icon="onDevice" />
+          {{ addedIndicator$() }}
+        </template>
+
+        <KButton
+          v-if="isSelected"
+          :text="coreString('removeAction')"
+          :primary="true"
+          @click="removeResource()"
+        />
+        <KButton
+          v-else
+          :text="addText$()"
+          :primary="false"
+          @click="addResource()"
+        />
+      </div>
+    </div>
+
+    <ResourceSelectionBreadcrumbs
+      v-if="ancestors.length"
+      :ancestors="[...ancestors, content]"
+      :channelsLink="channelsLink"
+      :topicsLink="topicsLink"
+      class="align-breadcrumbs"
+    />
+
+    <div class="title-class">
+      <h5>
```

Comment:
Also wonder if this tag should be an `h5`

Reply:
I think when you can see it here once you double click it 
https://www.figma.com/design/lVJt5ukOrxS9qay0rXy7GC/0.18-Coach-updates?node-id=2456-32151&t=ymu9hEzTQkS2oKOP-0